spec/services/integration_tests/test_runner_spec.rb

```diff
@@ -18,12 +18,35 @@ def post_resource(url, params)
       expect(JSON.parse(response.body)['success']).to eq true
     end
 
+    def fetch_result(assessment_id)
+      get assessment_path(assessment_id), headers: headers
+      JSON.parse(response.body, symbolize_names: true)
+    end
+
+    def compare_result(assessment_id, payload, spreadsheet_name) # rubocop:disable Lint/UnusedMethodArgument
+      result = fetch_result(assessment_id) # rubocop:disable Lint/UselessAssignment
+
+      # TODO: uncomment to see difference between actual and expected results
+      # p '---------'
+      # p spreadsheet_name
+      # p 'actual <-> expected'
+      # p "#{result[:assessment_result]} <-> #{payload[:expected_results][:overall_result]}"
+      # p "total_capital : #{result[:capital][:total_capital]} <-> #{payload[:expected_results][:applicant_disposable_capital]}"
+      # p "capital_contribution: #{result[:capital][:capital_contribution]} <-> #{payload[:contribution_results][:from_capital]}"
```

Comment:
I wonder if a better way to handle this would be to create a verbose output something like:
```ruby
def verbose_output(assessment_id, payload, spreadsheet_name) 
  result = fetch_result(assessment_id)

  <<TEXT
  ---------
  #{spreadsheet_name}
  actual <-> expected
  #{result[:assessment_result]} <-> #{payload[:expected_results][:overall_result]}
  total_capital : #{result[:capital][:total_capital]} <-> #{ payload[:expected_results][:applicant_disposable_capital]}
  capital_contribution: #{result[:capital][:capital_contribution]} <-> #{ payload[:contribution_results][:from_capital]}
  TEXT
end
```
Then here add the line:
```ruby
  puts verbose_output(assessment_id, payload, spreadsheet_name) if ENV['VERBOSE'] == 'true'
```
Then to display the extra output you do:
```bash
VERBOSE=true rspec spec/services/integration_tests/test_runner_spec.rb
```